library/Rules/PostalCode.php

```diff
@@ -122,7 +122,7 @@ class PostalCode extends Regex
         'NF' => "/^(\d{4})$/",
         'NG' => "/^(\d{6})$/",
         'NI' => "/^(\d{7})$/",
-        'NL' => "/^(\d{4}[A-Z]{2})$/",
+        'NL' => "/^(\d{4}\s?[A-Z]{2})$/",
```

Comment:
`\s` stands for `\t` and `\n` as  well. I believe only a single space is what you want